plugins/polio/js/src/domains/VaccineModule/SupplyChain/Details/shared.tsx

```diff
@@ -62,7 +63,9 @@ export const usePaperStyles = makeStyles((theme: Theme) => ({
         '&  label.MuiInputLabel-root': {
             backgroundColor: `${theme.palette.grey['200']} !important`,
         },
-        backgroundColor: theme.palette.grey['200'],
+        // Couldn't apply the color with bgColor and sx. Also theme.palette.grey['200'] woudln't work either
+        backgroundColor: `${theme.palette.grey['200']} !important`,
```

Comment:
Yes you can use theme.palette.grey['200'] in sx prop:

`sx={{ backgroundColor: theme => theme.palette.grey[200] }}`

Reply:
Nope, I tried, it didn't work